rock_paper_scissors/rps.py

```diff
@@ -3,12 +3,24 @@
 import sys
 
 def rock_paper_scissors(n):
-  pass 
+    
```

Comment:
really nice solution too! I just solved this on a javascript code challenge, and my solution doesn't come close to this. very, very nice use of recursion.